src/foraging_gui/MyFunctions.py

```diff
@@ -961,7 +965,7 @@ def _GetLaserWaveForm(self):
         self.CLP_OffsetEnd=float(eval('self.TP_OffsetEnd_'+N)) # negative, backward; positive forward
         self.CLP_SampleFrequency=float(self.TP_SampleFrequency)
         # align to trial start
-        if (self.CLP_LaserStart=='Trial start' or self.CLP_LaserStart=='Go cue') and self.CLP_LaserEnd=='NA':
+        if (self.CLP_LaserStart=='Trial start' or self.CLP_LaserStart=='Go cue' or self.CLP_LaserStart=='Reward outcome') and self.CLP_LaserEnd=='NA':
```

Comment:
Is there documentation somewhere about these keywords for `CLP_LaserStart`? If so, did we add something about `Reward outcome'? If not, we should add some documentation and add all these keywords.

Reply:
I documented the Reward outcome in the documentation.